src/XPlot.Plotly.Interactive/XPlot.Plotly.Interactive.fsproj

```diff
@@ -1,48 +1,52 @@
-  </ItemGroup>
-
-  <ItemGroup>
-    <None Include="$(OutputPath)/XPlot.Plotly.Interactive.dll" Pack="true" PackagePath="interactive-extensions/dotnet" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <ProjectReference Include="../XPlot.Plotly/XPlot.Plotly.fsproj"/>
-  </ItemGroup>
-
-  <ItemGroup>
-    <None Include="..\..\docs\img\logo.png" Pack="true" PackagePath="\" />
-  </ItemGroup>
+    <PropertyGroup>
+        <OutputType>Library</OutputType>
+        <TargetFramework>net7.0</TargetFramework>
+        <PackageId>XPlot.Plotly.Interactive</PackageId>
+        <IsPackable>true</IsPackable>
+        <PackageVersion>4.0.7</PackageVersion>
+        <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
+        <RepositoryType>git</RepositoryType>
+        <PackageProjectUrl>https://fslab.org/XPlot/</PackageProjectUrl>
+        <RepositoryUrl>https://github.com/fslaborg/XPlot</RepositoryUrl>
+        <Authors>fsproject contributors</Authors>
+        <Description>Lets you use XPlot.Plotly to be used in .NET Interactive. When you use this package, you can embed Plotly charts directly into a .NET Interactive notebook!</Description>
+        <PackageIcon>logo.png</PackageIcon>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <Compile Include="NewPlotlyChartCommand.fs" />
+        <Compile Include="KernelExtension.fs" />
+        <Compile Include="XPlotExtensions.fs" />
+    </ItemGroup>
+
+    <ItemGroup>
+        <PackageReference Include="Giraffe.ViewEngine" Version="1.2.0" />
+        <PackageReference Include="Microsoft.DotNet.Interactive" Version="1.0.0-beta.23562.1" />
+        <PackageReference Include="Microsoft.DotNet.Interactive.Formatting" Version="1.0.0-beta.23562.1" />
+        <PackageReference Include="Microsoft.DotNet.Interactive.PowerShell" Version="1.0.0-beta.23562.1" />
+    </ItemGroup>
+
+    <ItemGroup>
+        <Content Include="$(MSBuildThisFileDirectory)Modules\**" Link="Modules\%(RecursiveDir)%(FileName)%(Extension)" PackagePath="contentFiles/any/any/Modules" PackageCopyToOutput="true" CopyToOutputDirectory="PreserveNewest" />
+        <Content Include="$(MSBuildThisFileDirectory)Modules\**" Link="Modules\%(RecursiveDir)%(FileName)%(Extension)" PackagePath="interactive-extensions/dotnet/Modules" PackageCopyToOutput="true" CopyToOutputDirectory="PreserveNewest" />
+    </ItemGroup>
+
+    <ItemGroup>
+        <None Include="extension.dib" Pack="true" PackagePath="interactive-extensions/dotnet" />
+    </ItemGroup>
+
+    <ItemGroup>
+        <ProjectReference Include="../XPlot.Plotly/XPlot.Plotly.fsproj" />
+    </ItemGroup>
+
+    <ItemGroup>
+        <None Include="..\..\docs\img\logo.png" Pack="true" PackagePath="\" />
+    </ItemGroup>
+
+    <ItemGroup>
+      <PackageReference Update="FSharp.Core" Version="8.0.100" />
+    </ItemGroup>
```

Comment:
I think we should also not pull in latest FSharp.Core here, esp. since it's an older TFM (.NET 7)